scripts/servers.py

```diff
@@ -757,14 +759,19 @@ def managed_acceptance_tests_server(
         Exception. The suite_name is not in the list of the acceptance tests
             suite names.
     """
-    if suite_name not in common.ACCEPTANCE_TESTS_SUITE_NAMES:
+    if not mobile and suite_name not in common.ACCEPTANCE_TESTS_SUITE_NAMES:
         raise Exception('Invalid suite name: %s' % suite_name)
 
     if headless:
         os.environ['HEADLESS'] = 'true'
     else:
         os.environ['HEADLESS'] = 'false'
 
+    if mobile:
```

Comment:
Here and above, can do: `os.environ['MOBILE'] = 'true' if mobile else 'false'`

Reply:
Done